lib/plugins/helper/toc.js

```diff
@@ -3,12 +3,13 @@
 const { tocObj, escapeHTML, encodeURL } = require('hexo-util');
 
 function tocHelper(str, options = {}) {
-  options = Object.assign({
+  options = {
```

Comment:
Only when first parameter of `Object.assign` is an empty Object will syntax be faster than `Object.assign`. Otherwise I can't see why should we use spread syntax to shallow merge Object.